core/src/models/GrouparooRecord.ts

```diff
@@ -29,7 +29,7 @@ import { Source } from "./Source";
 import { GrouparooModel } from "./GrouparooModel";
 import { ModelGuard } from "../modules/modelGuard";
 
-const STATES = ["draft", "pending", "ready", "deleted"] as const;
+const STATES = ["draft", "pending", "exporting", "ready", "deleted"] as const;
```

Comment:
Alternatively to adding a new state, we could make a new column on the record `readyForExport (bool)`